### DISCUSSION TITLE
K factor  LA

### DISCUSSION THREAD

**Discussion by Cthutquz** [supermerill/SuperSlicer#1724] [Community contribution]
In the Calibrate menu, add LA ( K-Factor) Calibration. Now without entry in Start G-Code M900 K0.08 ( for ЗУЕП )from the site of the original Prusa, printing test cube is very poor quality. After entering Filament Start G-Code M900 k0.08 for PETG, we get a great result. A model for calibrating and finding the correct value of K-Factor is required. Or make a menu item in the printer settings where you can prescribe the value of K Factor

**Comment by tome9111991** [comment]
Hello
You can use https://marlinfw.org/tools/lin_advance/k-factor.html ?

**Comment by supermerill** [comment]
Duplicate of #320 ?
please use the issue to centralize the information
